src/generic/treectlg.cpp

```diff
@@ -2381,11 +2381,13 @@ void wxGenericTreeCtrl::CalculateLineHeight()
         // Calculate a m_lineHeight value from the normal Image sizes.
         // May be toggle off. Then wxGenericTreeCtrl will spread when
         // necessary (which might look ugly).
-        int n = GetImageCount();
-        for (int i = 0; i < n ; i++)
+        auto &imgs = GetImages();
+        for ( auto &img : imgs )
```

Comment:
Sorry, but why did this have to be changed?

I'm afraid it's wrong because it breaks compatibility for code using `wxImageList`: `GetImages()` will be empty in this case, so the size of the images in the list won't be taken into account, unlike before.

Reply:
Okay, got it. I reverted those lines.